content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,118 @@
+================
+Google Translate
+================
+
+*Google Translate* can be used to translate user generated text in the Odoo chatter. To translate a
+user's text from another language click the *three dot* menu to the right of the chatter. Then
```

Comment:
I think *three dot* needs a guilabel, also I would confirm the actual name in the inspect element menu. (I believe it is *ellipsis* but I am not sure, let me know and I will add it to the style guide doc)

Reply:
You're right it is an ellipsis-- but I fear our customers won't understand this phrase. I'm going to put in parentheses.